<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html dir="ltr" lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <link href="../misc/favicon.ico" rel="shortcut icon" type="image/x-icon"/>
  <title>
   phylogenetic methods 2013 | mtholder.github.io/home/   </title>
  <link href="../modules/book/book.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../modules/node/node.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../modules/system/defaults.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../modules/system/system.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../modules/system/system-menus.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../modules/user/user.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/cck/theme/content-module.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/date/date.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/date/date_popup/themes/datepicker.1.7.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/date/date_popup/themes/jquery.timeentry.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/geshifilter/geshifilter.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/biblio/biblio.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/cck/modules/fieldgroup/fieldgroup.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../sites/all/modules/views/css/views.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../themes/garland/style.css?F" media="all" rel="stylesheet" type="text/css"/>
  <link href="../themes/garland/print.css?F" media="print" rel="stylesheet" type="text/css"/>
  <!--[if lt IE 7]>
      <link type="text/css" rel="stylesheet" media="all" href="/themes/garland/fix-ie.css" />    <![endif]-->
 </head>
 <body class="sidebar-left">
  <!-- Layout -->
  <div class="clear-block" id="header-region">
  </div>
  <div id="wrapper">
   <div class="clear-block" id="container">
    <div id="header">
     <div id="logo-floater">
      <h1>
       <a href="../" title="mtholder.github.io/home/">
        <img alt="mtholder.github.io/home/" id="logo" src="../sites/default/files/garland_logo.png"/>
        <span>
         mtholder.github.io/home/         </span>
       </a>
      </h1>
     </div>
     <ul class="links primary-links">
      <li class="menu-440 first">
       <a href="../home" title="Holder Lab Web page">
        Home
       </a>
      </li>
      <li class="menu-449">
       <a href="../courses" title="Courses">
        Courses
       </a>
      </li>
      <li class="menu-628 last">
       <a href="https://mtholder.github.io/home/content/software.html" title="Software">
        Software
       </a>
      </li>
     </ul>
    </div>
    <!-- /header -->
    <div id="center">
     <div id="squeeze">
      <div class="right-corner">
       <div class="left-corner">
        <div class="breadcrumb">
         <a href="../">
          Home
         </a>
        </div>
        <h2>
         phylogenetic methods 2013
        </h2>
        <div class="clear-block">
         <div class="node" id="node-718">
          <div class="content clear-block">
           <p>
            <strong>
             Note:
            </strong>
            Note that the current version of the course is at
            <a href="https://mtholder.github.io/home/courses/phylomethods">
             "https://mtholder.github.io/home/courses/phylomethods
            </a>
           </p>
           <p>
            This course covers state-of-the-art methods for reconstructing phylogenies. We will cover the theoretical basis for different phylogenetic analyses and learn how to use some of the software packages available for conducting these analyses. Inferences that rely heavily on phylogenetic trees (
            <em>
             eg.
            </em>
            analyses of character evolution, divergence time estimation, and studies of diversification rates) will also be covered.
           </p>
           <p>
            The syllabus is available as a
            <a href="https://mtholder.github.io/home/sites/default/files/BIOL-848-2013-Syllabus.pdf">
             pdf from this link
            </a>
            .
           </p>
           <p>
            I will post pdfs here throughout the semester. I will post the source code for my own notes and some slides to a
            <a href="http://github.com/mtholder/PhylogeneticMethodsCourse">
             GitHub repository
            </a>
            , but much of the lecture is presented on the board. Bear in mind, that a great deal of the content will be given on the board, and my typed notes may be minimal for those sections of the lectures - make sure that you take good notes!
           </p>
           <h2>
            Some links
           </h2>
           <p>
            Felsenstein's
            <a href="http://www.sinauer.com/detail.php?id=1775">
             Inferring Phylogenies
            </a>
            book is highly recommended for the course.
           </p>
           <p>
            His
            <a href="http://evolution.gs.washington.edu/phylip/software.html">
             list of phylogeny programs
            </a>
            is also very helpful.
           </p>
           <p>
            The website for the
            <a href="https://molevolworkshop.github.io/">
             Workshops on Molecular Evolution
            </a>
            is a good source of lecture slides, software tutorials, and general information.
           </p>
           <h2>
            Slides
           </h2>
           <table border="1">
            <tr>
             <th>
              Date
             </th>
             <th>
              Link
             </th>
            </tr>
            <tr>
             <td>
              Jan 23
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec1-IntroStats.pdf">
               lec1-IntroStats.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Jan 25
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec2-tree-terms.pdf">
               lec2-tree-terms.pdf
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/homework-2.pdf">
               Homework 2
              </a>
              (due on Wed)
             </td>
            </tr>
            <tr>
             <td>
              Jan 28
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec3-ParalogyAndInference.pdf">
               lec3-ParalogyAndInference.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Jan 30
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec4-Inference.pdf">
               lec4-Inference.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 1
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/BIOL848-lab1-UnixIntro.html">
               lab 1 Unix terminal
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/BIOL848-lec5-StatInference-Notes.pdf">
               BIOL848-lec5-StatInference-Notes.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 4
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec6-ML-CompatModel-Notes.pdf">
               lec6-ML-CompatModel-Notes.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 6
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec7-ML-MPModel-Notes.pdf">
               lec6-ML-MPModel-Notes.pdf
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/lec7-ML-MPModelAlgorithms.pdf">
               lec6-ML-MPModelAlgorithms.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 8
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lab2-PaupMesquiteNEXUS/lab2-paupNEXUSMesquite.html">
               lab 2-PaupMesquiteNEXUS
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 18
             </td>
             <td>
              Paul Lewis' parsimony slides are at
              <a href="https://mtholder.github.io/home/slides/POL-ParsSlides.pdf">
               POL-ParsSlides.pdf
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/BIOL848-lec8-Searching.pdf">
               BIOL848-lec8-Searching.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 25
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/parsSummaryModels.pdf">
               parsSummaryModels.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Feb 27
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/BIOL848-lec11ML.pdf">
               BIOL848-lec11ML.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 1
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lab3-PaupSearching/index.html">
               lab 3
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/lab5ML/lab5ML.html">
               lab 4
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 4
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/treeLikeRateHetIntro.pdf">
               treeLikeRateHetIntro.pdf
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/BIOL848-Homework3.pdf">
               Homework 3
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 6
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/BIOL848-rateHetContinued.pdf">
               BIOL848-rateHetContinued.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 8
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lab5ML/lab5ML.html">
               lab 5 (same as last week's lab 4)
              </a>
              and
              <a href="https://mtholder.github.io/home/slides/GarliDemo/garliExerciseDC.html">
               a GARLI lab
              </a>
              <br/>
              <br/>
              <a href="https://mtholder.github.io/home/slides/basicNucleotideGarli.txt">
               basicNucleotideGarli.txt
              </a>
              <br/>
              <br/>
              <a href="https://mtholder.github.io/home/slides/constrainedNucleotideGarli.txt">
               constrainedNucleotideGarli.txt
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 11
             </td>
             <td>
              <a href="https://mtholder.github.io/home/woodshole/HolderTestingLectureWoodsHole2012.pdf">
               HolderTestingLectureWoodsHole2012.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Mar 15
             </td>
             <td>
              <a href="https://molevol.mbl.edu/wiki/index.php/ParametricBootstrappingLab">
               Parametric Bootstrapping Lab
              </a>
              <br/>
              <a href="https://mtholder.github.io/home/slides/seq-gen.zip">
               seq-gen.zip for Windows
              </a>
              which requires the CygWin DLL from
              <a href="http://www.cygwin.com/">
               cygwin
              </a>
              be in the same directory as the executable.
              <br/>
              <a href="https://mtholder.github.io/home/slides/simdata.nex">
               simdata.nex
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Week of Mar 25
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/BIOL848-BayesIntro.pdf">
               BIOL848-BayesIntro.pdf
              </a>
              <br/>
              <br/>
              <a href="https://mtholder.github.io/home/slides/BIOL848-BayesSecond.pdf">
               BIOL848-BayesSecond.pdf
              </a>
              <br/>
              <br/>
              Recommended Lab:
              <a href="http://bodegaphylo.wikispot.org/MrBayes_Tutorial_%28Brown%29">
               Jeremy Brown's MrBayes lab
              </a>
              <br/>
              Alternate Labs:
              <a href="https://molevol.mbl.edu/wiki/index.php/MrBayes">
               Woods Hole labs
              </a>
              <br/>
              Holder/Lewis old
              <a href="https://mtholder.github.io/home/slides/lab8-Bayesian/mblab.pdf">
               MrBayes Lab
              </a>
              . The algaemb datafile is
              <a href="https://mtholder.github.io/home/slides/lab8-Bayesian/algaemb.nex">
               here
              </a>
              .
              <br/>
              The jc.nex datafile is
              <a href="https://mtholder.github.io/home/slides/lab8-Bayesian/jc.nex">
               here
              </a>
              .
              <br/>
              The mac executable  is
              <a href="https://mtholder.github.io/home/slides/lab8-Bayesian/mb-3.1.2-mac.zip">
               here
              </a>
              . An old version is
              <a href="https://mtholder.github.io/home/slides/lab8-Bayesian/mbdir.zip">
               here
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 1
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec18-BayesFactors.pdf">
               lec18-BayesFactors.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 3, 8
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec19-AncStates.pdf">
               lec19-AncStates.pdf
              </a>
              (corrected Apr 5)
             </td>
            </tr>
            <tr>
             <td>
              Apr 5
             </td>
             <td>
              See Jeremy Brown's MrBayes lab
              <a href="http://bodegaphylo.wikispot.org/MrBayes_Tutorial_%28Brown%29?action=Files&amp;do=view&amp;target=MrBayes_tutorial.zip">
               MrBayes_tutorial.zip
              </a>
              Some background on the MrBayes labs from woods hole:
              <a href="https://mtholder.github.io/home/slides/MrBayes_tutorial.txt">
               MrBayes_tutorial.txt
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 10
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec20-CharEvol.pdf">
               lec20-CharEvol.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 11
             </td>
             <td>
              Your choice of
              <a href="https://mtholder.github.io/home/slides/lab9-MesquiteAncStatesBisse/09-Mesquite.html">
               a Mesquite lab on character state estimation
              </a>
              <br/>
              or you can try out GARLI's
              <a href="https://github.com/zwickl/garli/wiki/Garli_Mkv_morphology_model">
               impmentation of the Mk model
              </a>
              , or
              <br/>
              See the tutorial in the Manual (PDF) for
              <a href="http://www.evolution.reading.ac.uk/BayesTraits.html">
               BayesTraits
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 10-17
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec22-MorphologicalCharacters.pdf">
               lec22-MorphologicalCharacters.pdf
              </a>
              <br/>
              <a href="https://molevol.mbl.edu/wiki/images/6/6f/Bodega_2013_divtime_lecture.pdf">
               Tracy Heath's div times slides
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 19
             </td>
             <td>
              <a href="http://treethinkers.org/divergence-time-estimation-using-beast/">
               Tracy Heath's BEAST lab
              </a>
              and (mainly Joe Felsenstein's) slides on the coalescent
              <a href="https://mtholder.github.io/home/slides/lec23-Coalescent.pdf">
               lec23-Coalescent.pdf
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 10-17
             </td>
             <td>
              I'll use some slides from Joseph Heled's
              <a href="https://docs.google.com/file/d/0BxbPcmfTXtMiamFRdjdEX3lZODQ/edit?usp=sharing">
               Feb 2013 talk in Austin
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 26
             </td>
             <td>
              I'll use some slides from Joseph Heled's
              <a href="https://docs.google.com/folder/d/0BxbPcmfTXtMiVkJteGdMYzhDeWM/edit">
               *BEAST tutorial
              </a>
             </td>
            </tr>
            <tr>
             <td>
              Apr 29
             </td>
             <td>
              <a href="https://mtholder.github.io/home/slides/lec24-MSA.pdf">
               lec24-MSA.pdf
              </a>
              , and I'll use some slides from Joe Bielawski's lecture on codon models from
              <a href="http://myweb.dal.ca/js551958/PAML_lab/lecture_slides.html">
               Joe's site
              </a>
             </td>
            </tr>
            <tr>
             <td>
              May 3
             </td>
             <td>
              lab. Check out either the RAxML lab at
              <a href="http://sco.h-its.org/exelixis/hands-On.html" title="http://sco.h-its.org/exelixis/hands-On.html">
               http://sco.h-its.org/exelixis/hands-On.html
              </a>
              <p>
               or the Joe Bielawski's PAML lab posted at
               <a href="http://myweb.dal.ca/js551958/PAML_lab/lab.html" title="http://myweb.dal.ca/js551958/PAML_lab/lab.html">
                http://myweb.dal.ca/js551958/PAML_lab/lab.html
               </a>
              </p>
             </td>
            </tr>
           </table>
          </div>
         </div>
        </div>
       </div>
      </div>
      <div class="clear-block">
       <div class="meta">
       </div>
      </div>
      <div id="footer">
       <p>
        The
        <a href="https://mtholder.github.io/home/">
         Holder Lab
        </a>
        is part of the
        <a href="https://eeb.ku.edu/">
         Department of Ecology and Evolutionary Biology
        </a>
        at
        <a href="https://www.ku.edu">
         KU
        </a>
       </p>
      </div>
      <!-- /.left-corner, /.right-corner, /#squeeze, /#center -->
      <!-- /container -->
      <!-- /layout -->
     </div>
    </div>
   </div>
  </div>
 </body>
</html>
